src/phpDocumentor/Dsn.php

```diff
@@ -64,15 +62,15 @@ final class Dsn
      */
     public function __construct(UriInterface $uri, array $parameters, string $dsn)
     {
-        $this->dsn = $dsn;
+        $this->dsn        = $dsn;
         $this->parameters = $parameters;
-        $this->uri = $uri;
+        $this->uri        = $uri;
     }
 
     public static function createFromString(string $dsn) : self
     {
-        $parameters = explode(';', $dsn);
-        $uri = self::parseUri(array_shift($parameters));
+        $parameters       = explode(';', $dsn);
+        $uri              = UriFactory::createUri(array_shift($parameters));
```

Comment:
Maybe we should standardize whether to align or not to prevent back and forth changes :)